app/presenters/oregon_digital/iiif_presenter.rb

```diff
@@ -18,21 +18,31 @@ def manifest_metadata
     end
 
     # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength
+    # TODO: DRY this up with #work_presenters?
     def file_set_presenters
-      presenters = []
-      file_sets.each do |fs|
+      file_set_ids = (ordered_ids & file_sets.map(&:id))
```

Comment:
I wonder if we need the ordered_ids here. Since its just filesets, we should be able to get away with just those. Is there a reason why we would need to grab just filesets that are included in ordered_ids as well?

Reply:
I didn't check to see if `file_set_ids` order actually updates when they're re-arranged. If they are then we should be able to get away with not using `ordered_ids`, otherwise I think it's important

Reply:
```
>> file_sets.map(&:id)
=> ["f0n009w228g", "f2n009w228g", "f4n009w228g", "f6n009w228g", "f8n009w228g", "f10n009w228g"]

>> ordered_ids & file_sets.map(&:id)
=> ["f8n009w228g", "f4n009w228g", "f0n009w228g", "f2n009w228g", "f6n009w228g", "f10n009w228g"]

```

`ordered_ids & file_sets.map(&:id)` is the correct order